packages/node_modules/brightspace-auth-keys-dynamodb-store/src/public-key-table-setup.js

```diff
@@ -0,0 +1,50 @@
+'use strict';
+
+function createPublicKeyTable(db, tablename, readCapacity, writeCapacity) {
+	var params = {
+		AttributeDefinitions: [
+			{
+				AttributeName: 'PublicKey',
+				AttributeType: 'S'
+			}
+		],
+		KeySchema: [
+			{
+				AttributeName: 'PublicKey',
+				KeyType: 'HASH'
+			}
+		],
+		ProvisionedThroughput: {
+			ReadCapacityUnits: readCapacity || 5,
+			WriteCapacityUnits: writeCapacity || 5
+		},
+		TableName: tablename
+	};
+
+	return db.createTable(params).promise().then(() => {
+		return setTtl(db, tablename);
+	}).catch((e) => {
+		// ResourceInUseException indicates the table already exists

```

Comment:
Assuming that the ttl is also set if the table already exists?

Reply:
Or that if they manually created that they decided they didn't want it for whatever reason